src/NUnitFramework/framework/Internal/Builders/NUnitTestFixtureBuilder.cs

```diff
@@ -97,7 +97,7 @@ public TestSuite BuildFrom(ITypeInfo typeInfo, ITestFixtureData testFixtureData)
             if (typeInfo.ContainsGenericParameters)
             {
                 Type[] typeArgs = testFixtureData.TypeArgs;
-                if (typeArgs.Length == 0)
+                if (typeArgs == null || typeArgs.Length == 0)
```

Comment:
This part of the PR looks good to me. 👍

Reply:
Everything else in the PR is tests, which also look good to me so we're good! 😃